From 8693eee5c69f056158283afdb9ba925f484f9acf Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 29 Mar 2022 08:46:10 +0800
Subject: [PATCH] fix python 3.10 compatibility

Fix following warning:
DeprecationWarning: There is no current event loop

Since Python 3.10 asyncio.get_event_loop() is deprecated, and
in the future, this function will be an alias of get_running_loop().

Upstream-Status: Pending

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 fullmetalupdate.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/fullmetalupdate.py b/fullmetalupdate.py
index cc416d4..95f20f5 100755
--- a/fullmetalupdate.py
+++ b/fullmetalupdate.py
@@ -94,5 +94,4 @@ async def main():
 
 if __name__ == '__main__':
     # create event loop, open aiohttp client session and start polling
-    loop = asyncio.get_event_loop()
-    loop.run_until_complete(main())
+    asyncio.run(main())
-- 
2.25.1

